scss/_reboot.scss

```diff
@@ -366,8 +366,8 @@ sup { top: -.5em; }
     color: var(--#{$prefix}link-visited-color);
   }
 
-  &:focus-visible,
-  &:focus[data-focus-visible-added] {
+  &:focus-visible:not([aria-disabled="true"]),
+  &:focus[data-focus-visible-added]:not([aria-disabled="true"]) {
```

Comment:
Need to decide whether we keep focus visible polyfill or not, but it might simplfied if not. Keep it in mind for later